src/applications/rated-disabilities/components/ServerError.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+export default function ServerError() {
+  return (
+    <va-alert status="error" uswds>
```

Comment:
if we are adding a new va-alert to rated disabilities that has uswds enabled then we will want to update the ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/76124 and spreadsheet [here](https://docs.google.com/spreadsheets/d/1ajKjN6dr-2PzpgEbfX1OqxAupvbqoHwqh7ml1m5a1Fo/edit?usp=sharing) to make sure we account for this work.

Reply:
Added to spreadsheet